doc/telescope.txt

```diff
@@ -257,6 +257,15 @@ actions.open_qflist()                                  *actions.open_qflist()*
 
 
 
+actions.delete_buffer({prompt_bufnr})                *actions.delete_buffer()*
```

Comment:
Is there a chance your fork has workflows disabled? Because the CI should handle the generation. We are also hitting a max query result issue for this file (which would remove another documentation for a function see https://github.com/tjdevries/tree-sitter-lua/issues/5 and all linked threads).

So other actions only use `--` spaces (that documentation will not be exported) (example `actions.git_staging_toggle`) to avoid this issue

Reply:
I do have workflows enabled but it did not kick in. It might be something related to GitHub's new feature where a user with privileged rights will have to approve the workflow runs for a first-time contributor: https://github.blog/2021-04-22-github-actions-update-helping-maintainers-combat-bad-actors/

> So other actions only use -- spaces (that documentation will not be exported) (example actions.git_staging_toggle) to avoid this issue

Got it, I will update the function doc but do we want this action as global or local to just `Telescope  buffers` as done in my first commit?

Reply:
Nevermind, the workflows were enabled in the repo but I had to give my explicit permission. Sorry for all the trouble :)